futureSquash/src/main/scala/com/softwaremill/futuresquash/FutureSquash.scala

```diff
@@ -0,0 +1,46 @@
+package com.softwaremill.futuresquash
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util._
+
+object FutureSquash {
+
+  /**
+    * Converts an Either[Throwable, A] to a Future[A] that may raise a Throwable
+    *
+    * @param either
```

Comment:
This part of documentation isn't useful. Could leave just the description above?